stringfunc.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+import "strings"
+
+// StringFunc - string manipulation function wrappers
+type StringFunc struct {}
+
+func (t StringFunc) Replace(s, old, new string) string {
```

Comment:
I think all of this has been addressed with the latest push - I'll wait to watch CI complete now I'm not in a plane ;-)